### PR TITLE
Remove gradle-wrapper.

### PR DESCRIPTION
**Pull request by godofredoc**
Gradle-wrapper is downloaed automatically by the flutter tool. The
pattern was also added to .gitignore to prevent from checking it in
again.

Bug: https://github.com/flutter/flutter-intellij/issues/6072

## Pre-launch Checklist

- [X] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [X] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [X] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [X] I signed the [CLA].
- [X] I listed at least one issue that this PR fixes in the description above.
- [X] I updated/added relevant documentation (doc comments with `///`).
- [X] I added new tests to check the change I am making, or this PR is [test-exempt].
- [X] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -36,3 +36,4 @@ flutter-studio/.gradle/
 flutter-idea/build/
 flutter-idea/.gradle/
 tool/plugin/.idea
+**/gradle-wrapper.jar
```
Comment (stevemessick) [inline .gitignore:39]: These Gradle wrappers are not downloaded by the Flutter tool. They are used to maintain the repo.

I'm creating `.allstar/binary_artifacts.yaml` to address this issue. I'm documenting each artifact we require.

**Review by stevemessick — CHANGES_REQUESTED** [review]
I think you can close this, unless using the yaml file is not permitted.

**Comment by stevemessick** [comment]
Hixie provided clarification as to how we need to handle this. I'll make the changes. But we will preserve the Gradle wrappers, so this PR won't be needed. Thanks for the heads-up :)

**Comment by godofredoc** [comment]
> Hixie provided clarification as to how we need to handle this. I'll make the changes. But we will preserve the Gradle wrappers, so this PR won't be needed. Thanks for the heads-up :)

Thank you. Closing this PR.

**Pull request closed without merging**